src/drive/web/modules/actions/utils.js

```diff
@@ -122,7 +122,7 @@ export const exportFilesNative = async (client, files, filename) => {
   const downloadAllFiles = files.map(async file => {
     const response = await client
       .collection('io.cozy.files')
-      .fetchFileContent(file.id)
+      .fetchFileContent(file.id || file._id)
```

Comment:
Are we sure we need both ? Why ?

Reply:
We need both since https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#iocozyfilesversions doesn't have an `id`. 

You remind me that cozy-client should add this `id` by itself, no (during this normalization process)? So I don't know why :
```jsx 
client
                .all('io.cozy.files.versions')
                .where({
                  relationships: { file: { data: { _id: fileId } } },
                  updated_at: { $gt: null }
                })
                .sortBy([
                  { 'relationships.file.data._id': 'desc' },
                  { updated_at: 'desc' }
                ])
                .indexFields(['relationships.file.data._id', 'updated_at'])
            }
```

is not returning an id.

issue created on #812

Reply:
Can't we only use '_id' here ?

Reply:
I'll fix the issue on CozyClient side and remove the need of both attribute. I prefer to do that instead of relying on _id which we avoid, right?

Reply:
Nevermind 49a3d438d